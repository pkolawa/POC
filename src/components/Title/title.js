import React from "react";
import Typography from "@material-ui/core/Typography";

class Title extends React.Component {
  render() {
    return (
      <Typography
        align="center"
        variant="h2"
        color={this.props.color ? this.props.color : "secondary"}
      >
        {this.props.title}
      </Typography>
    );
  }
}

export default Title;
