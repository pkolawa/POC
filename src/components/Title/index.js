import title from "./title";

export { title as default };