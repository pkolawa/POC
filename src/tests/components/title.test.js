import React from "react";
import { shallow } from "enzyme";
import Title from "../../components/Title/title";
test("Should Render title", () => {
  const wrapper = shallow(<Title title={"App Console"}/>);
  expect(wrapper).toMatchSnapshot();
});