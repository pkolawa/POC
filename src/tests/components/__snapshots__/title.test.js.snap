// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render title 1`] = `
<WithStyles(ForwardRef(Typography))
  align="center"
  color="secondary"
  variant="h2"
>
  App Console
</WithStyles(ForwardRef(Typography))>
`;
